lib/core/elastic_connector_actions.rb

```diff
@@ -132,22 +132,30 @@ def update_connector_status(connector_id, status, error_message = nil)
         update_connector_fields(connector_id, body)
       end
 
-      def complete_sync(connector_id, job_id, status)
-        sync_status = status[:error] ? Connectors::SyncStatus::ERROR : Connectors::SyncStatus::COMPLETED
+      def update_sync(job_id, metadata)
```

Comment:
`update_sync` method will only update the job index, and `last_indexed_document_count` and `last_deleted_document_count` are only updated in the connector index in `complete_sync` method. cc @sphilipse